src/App/Controller/PublicDiscussionController.php

```diff
@@ -230,6 +267,16 @@ public function deleteAction($slug)
                 'slug' => $proposalSlug,
             ]));
         }
+
+        if ($discussion->isArticleDiscussion()) {
```

Comment:
What if the controller does not go into this `if` ?

Reply:
An exception has been added after this if.